subprojects/junit-jupiter/src/test/java/org/mockitousage/StrictnessTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2018 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockitousage;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.engine.TestExecutionResult;
+import org.junit.platform.launcher.Launcher;
+import org.junit.platform.launcher.LauncherDiscoveryRequest;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestIdentifier;
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
+import org.junit.platform.launcher.core.LauncherFactory;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.exceptions.misusing.UnnecessaryStubbingException;
+import org.mockito.junit.jupiter.ConfiguredWithMockito;
+import org.mockito.quality.Strictness;
+
+import java.util.function.Function;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
+
+@SuppressWarnings("ConstantConditions")
+class StrictnessTest {
```

Comment:
How come this test does not need ```MockitoExtension```?

Reply:
Oh, meta-annotations. Cool.

Reply:
This test is intended to run a single JUnit test that then runs inner classes (which do have the extension)